easybuild/easyconfigs/i/ipyparallel/ipyparallel-6.2.2-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'ipyparallel'
+version = '6.2.2'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://ipyparallel.readthedocs.io'
+description = """ipyparallel is a Python package and collection
+ of CLI scripts for controlling clusters for Jupyter"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['02b225966d5c20f12b1fba0b6b10aa5d352a6b492e075f137ff0ff6e95b9358e']
+
+dependencies = [
+    ('Python', "3.6.4"),
+    ('IPython', '6.4.0', versionsuffix),
+]
+
+use_pip = True
+
+exts_download_dep_fail = True
```

Comment:
@SimonPinches This should be `download_dep_fail`, no extensions involved here...

Reply:
@boegel, fixed :-)